### ISSUE TITLE
Problem with new install 

### ISSUE THREAD

**Issue by skarpik** [nvisionative/nvQuickSite#359]
I'm trying to track down some bugs in 2sxc, so I thought I'd set up a DNN website on one of my Windows 10 computers (I've been using nvQuickSite and 2sxc on Windows 11).

I installed SQLExpress on my Windows 10 computer. Created an admin user. Then ran nvQuickSite and got this error:

![nvquicksite error](https://user-images.githubusercontent.com/3846223/184066094-de1d5486-8438-4a75-9dc8-e9975baf0e34.jpg)

Is there a bug or have I missed a step somewhere?

Thanks.

**Comment by david-poindexter** [comment]
@skarpik can you set logging to verbose in nvQuickSite and see if that provides any clues?

**Comment by skarpik** [comment]
Sure. Where do I turn the logging on?

**Comment by valadas** [comment]
You would have to first click on settings (orange bottom-left box)
![image](https://user-images.githubusercontent.com/6371568/184259217-d74f41f5-b82b-4a61-a2ea-21177dad3cd5.png)

And then you can raise the logging level and click the link to get to the log files after reproducing the issue.
![image](https://user-images.githubusercontent.com/6371568/184259268-4a9113cc-3590-40a4-8859-730744439e65.png)

**Comment by valadas** [comment]
Also, did you install IIS on that computer ?

**Comment by skarpik** [comment]
@valadas - thank you for pointing out the settings. I don't know how I missed that.

I have been busy on other github issues and just got back to this one this morning. I set things up to duplicate the error but this time with more diagnostics. And what happened? It worked perfectly. I then tried again with the regular level of logging. And it worked perfectly. It reminds me of what happens when you take your car in with the engine light on. As soon as the mechanic looks at the car the light goes off. I swear there is something similar happening here or I am going mad.

In any case, contrary to my earlier complaint, the software works perfectly. And as always is terribly useful. Sorry for the wild goose chase.
